lms/djangoapps/instructor_task/api.py

```diff
@@ -587,3 +587,37 @@ def process_scheduled_tasks():
             submit_scheduled_task(schedule)
         except QueueConnectionError as exc:
             log.error(f"Error processing scheduled task with task id '{schedule.task.id}': {exc}")
+
+
+def convert_schedule_to_utc_from_local(schedule, browser_timezone, user):
```

Comment:
This was originally a utility function of the `instructor` app that I moved this to the `instructor_task` app as it felt more appropriate with my recent changes. The `InstructorTaskSchedule` model and most of the functionality around the schedules exists in the `instructor_task` app. 

Functions that support schedules should be in this app. It is in the public Python API as the `instructor` app will leverage this functionality when we receive an initial scheduled email request via the Instructor Dashboard.